frontend/app/src/components/defi/loan/loans/LiquityLending.vue

```diff
@@ -69,29 +69,27 @@ const { tc } = useI18n();
           <loan-debt :debt="debt" :asset="debt.asset" />
         </v-col>
       </v-row>
-      <v-row no-gutters class="mt-8">
-        <v-col cols="12">
-          <premium-card
-            v-if="!premium"
-            :title="tc('liquity_lending.trove_events')"
-          />
-
-          <transaction-content
-            use-external-account-filter
-            :section-title="tc('liquity_lending.trove_events')"
-            :protocols="[TransactionEventProtocol.LIQUITY]"
-            :event-types="[
-              HistoryEventType.WITHDRAWAL,
-              HistoryEventType.SPEND,
-              HistoryEventType.DEPOSIT
-            ]"
-            :external-account-filter="{
+      <div v-if="!premium" class="mt-8">
```

Comment:
What was the problem with the old approach?

Reply:
:external-account-filter should send array instead

Reply:
I am talking about the v-row -> div change. Is it just removing vuetify dependencies?

Reply:
I think using <v-row> with only 1 <v-col> is showed the same with <div>

Reply:
> I think using `<v-row>` with only 1 `<v-col>` is look the same like `<div>`. Just want to remove unnecessary rendering.

It's not exactly the same because of the 12px gutter.